### ISSUE TITLE
Zoom and rotate on stat testing tab - dino graphic

### ISSUE THREAD

**Issue by litlhogn** [cadon/ARKStatsExtractor#1280]
The color picker and image is really nice.  It would be fantastic if you could zoom in on the image and rotate the dino image (in 3d?) for testing colors.

**Comment by cadon** [comment]
Yes, that would be nice indeed :) We thought about it and It's complicated. Maybe some day.
Do you have a specific species where the image is maybe not taken in a good angle? That could be fixed easier.

**Comment by litlhogn** [comment]
Yeah, figured it was pretty big ask.  Dinopithecus, for example, you can't really see the belly or eye colors.  Maybe add multiple picture angles on some of the more complex color dinos?  Was just a thought.

**Comment by cadon** [comment]
Yes, multiple angles was an option I thought about. Currently the images are all taken manually and with the amount of species that takes quite a while to take and process them.
I'll look into the image of the Dinopithecus at least to optimize that. If you find other species with not visible regions let me know.
